pipelines/deploy/CEPlugins/Jenkinsfile

```diff
@@ -35,7 +35,7 @@ node() {
                        mv content-plugins ansible
                     """
                     ansiblePlaybook = "${currentWs}/ansible/deploy-plugins.yml"
-                    ansibleExtraArgs = "--tags plugins --extra-vars \" source_file=${currentWs}/ansible/content-plugins  az_file_path=${currentWs}/ansible/content-plugins/az_copy.sh\" --vault-password-file /var/lib/jenkins/secrets/vault-pass"  
+                    ansibleExtraArgs = "--tags plugins --extra-vars \" source_file=${currentWs}/ansible/content-plugins s3_file_path=${currentWs}/ansible/content-plugins/s3_copy.sh  az_file_path=${currentWs}/ansible/content-plugins/az_copy.sh\" --vault-password-file /var/lib/jenkins/secrets/vault-pass"  
```

Comment:
Undo this file change. Revert to original state. This is generalized as part of https://github.com/project-sunbird/sunbird-devops/pull/3598/files#diff-76241b2a32c9e9467802eeb2188ca563245525bee7b119fd4d8da106b2796654R37

Reply:
reverted this change